### ISSUE TITLE
Parameter name mistmatch, may lead to confusion

### ISSUE THREAD

**Issue by NZSmartie** [Lobaro/lobaro-coap#3]
Saw this while working on multicast support. Will be providing a PR to fix this shortly. but making an issue just in case.

[interface/network/net_Endpoint.h:83](https://github.com/Lobaro/lobaro-coap/blob/master/interface/network/net_Endpoint.h#L83)
```c
void CopyEndpoints(NetEp_t* Source, NetEp_t* Destination);
```

[interface/network/net_Endpoint.c:55](https://github.com/Lobaro/lobaro-coap/blob/master/interface/network/net_Endpoint.c#L55-L58)
```c
void _rom CopyEndpoints(NetEp_t* Destination, NetEp_t* Source)
 {
	 memmove((void*)Destination, (void*)Source, sizeof(NetEp_t));
}
```

**Comment by Niondir** [comment]
Thanks, that makes sense.

I recommend working on the **api_refactoring branch.** We used that branch to fix a bunch of issues already and used to code in one project. So it's considered stable. Probably I will merge it to master soon. In any case I can wait for your Pull Request to avoid trouble :)

**Comment by NZSmartie** [comment]
This has been fixed in Pull Request #4
